cognite/client/_api/data_modeling/views.py

```diff
@@ -239,6 +239,65 @@ def apply(self, view: ViewApply | Sequence[ViewApply]) -> View | ViewList:
                 ...    )
                 ... ]
                 >>> res = c.data_modeling.views.apply(views)
+
+
+            Create views with edge relations::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes.data_modeling import (
+                ...     ContainerId,
+                ...     DirectRelationReference,
+                ...     MappedPropertyApply,
+                ...     ViewApply,
+                ...     ViewId,
+                >>> )
+                >>> from cognite.client.data_classes.data_modeling.views import (
+                ...     MultiEdgeConnectionApply,
```

Comment:
Lets just fix the `__init__` file in data modeling so that you can import `MultiEdgeConnectionApply` in the line above.

Reply:
done

Reply:
added the other dataclasses that seemed to not be included yet